supplier/migrations/0002_test_data.py

```diff
@@ -10,16 +10,17 @@ class Migration(migrations.Migration):
     def generate_data(apps, schema_editor):
         test_data = [
             ('worldMaster', 'ash', 'katchamp', 'pokemon123', 'pokemon inc'),
-            ('ed', 'edd', 'eddy', 'jawbreaker101', 'jawbreaker inc'),
+            ('ededed', 'edd', 'eddy', 'jawbreaker101', 'jawbreaker inc'),
```

Comment:
is there a reason for this change?

Reply:
min validator for this field is 6

Reply:
It was written false because the min validator is 6 and they were not using the supplier save function